RadarSDK/RadarLocationManager.m

```diff
@@ -1127,4 +1126,10 @@ - (void)locationManager:(CLLocationManager *)manager didFailWithError:(NSError *
     [self callCompletionHandlersWithStatus:RadarStatusErrorLocation location:nil];
 }
 
+- (void)locationManager:(CLLocationManager *)manager didChangeAuthorizationStatus:(CLAuthorizationStatus)status {
+    if ((status == kCLAuthorizationStatusAuthorizedAlways || status == kCLAuthorizationStatusAuthorizedWhenInUse) && [RadarSettings sdkConfiguration].trackOnceOnAppOpen) {
```

Comment:
I think we want `[RadarSettings sdkConfiguration].trackOnceOnAppOpen || [RadarSettings sdkConfiguration].startTrackingOnInitialize`